### DISCUSSION TITLE
When the "Format" Menu will be available?

### DISCUSSION THREAD

**Discussion by haplox** [twinbasic/twinbasic#1358] [General]
Hello,

I have searched this question in bugs, discussions and milestones, without to see when it is thought to implement it. Is there any planned date for this to be usefull?

Thanks

**Comment by fafalone** [comment]
Most of the options in the Format menu are already available. 

![image](https://user-images.githubusercontent.com/7834493/201819917-4e106626-e557-4b44-b050-3c485be63560.png)

The only ones I'm really missing are the 'make spacing equal' ones.

**Comment by haplox** [comment]
Yes, I know @fafalone , but the menu is there and there are some important options not accessible: align to the center (both vertical and horizontal), aling to the center of form, etc...

Thanks
